### PR TITLE
Added confidence and iterator functionality

### PR DESCRIPTION
**Pull request by tax**
Hello Greg,

I added some extra functions to this library with. I hope you have some time to take a look and let me know what you think.

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -0,0 +1,123 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+# Vagrantfile API/syntax version. Don't touch unless you know what you're doing!
+VAGRANTFILE_API_VERSION = "2"
+
+Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
+  # All Vagrant configuration is done here. The most common configuration
+  # options are documented and commented below. For a complete reference,
+  # please see the online documentation at vagrantup.com.
```
Comment (ryansb) [inline Vagrantfile]: I'm all for "comments in the code" but all these comments are just from the vagrant default template. Maybe trim this down a bit?

**Comment by tax** [comment]
Hi Ryan thanks for the feedback I removed the comments.

**Comment by gregjurman** [comment]
I would remove the FreeSansBold font file from the pull. 

There is probably a way to look for it locally on the end-user's machine rather than shipping a binary. Another sans-serif font could be subbed in if it is missing so having it is redundant.

**Comment by tax** [comment]
No problem font is now removed.

**Comment by gregjurman** [comment]
Ok everything looks good to me. Coverage reported a test missing for `get_textlines`. Other than that it looks good. Just add that test and I'll pull it in.

Thanks!

**Comment by tax** [comment]
Here you go

**Comment by gregjurman** [comment]
Thank You!

**Pull request merged**